### PR TITLE
Edit page fetch

### PR DESCRIPTION
**Pull request by rsidhi088**
Functionality to fetch details from the project panel page and edit them in the edit page.

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import WhyPerflow from "./components/WhyPerflow/WhyPerflow";
 import ProjPanel from "./components/Projects/ProjPanel";
 import EditProject from "./components/Projects/EditProject";
 
+
```
Comment (MelanyCristy) [inline frontend/src/App.js]: Blank space, please remove

File: frontend/src/App.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import WhyPerflow from "./components/WhyPerflow/WhyPerflow";
...
 import Navbar from "./components/Navbar";
```
Comment (MelanyCristy) [inline frontend/src/App.js]: Blank space, please remove

File: frontend/src/App.js [hunk 3]
```diff
@@ -36,7 +37,7 @@ class App extends React.Component {
   render() {
     const loggedInRouter = (
       <Router>
-        
+
```
Comment (MelanyCristy) [inline frontend/src/App.js]: Blank space, please remove

File: frontend/src/App.js [hunk 4]
```diff
@@ -48,7 +49,7 @@ class App extends React.Component {
               <Route path="/scurve" component={SCurve} />
             </Route>
           </Switch>
-       
+
```
Comment (MelanyCristy) [inline frontend/src/App.js:46]: Blank space, please remove

File: frontend/src/components/Projects/EditProject.js [hunk 5]
```diff
@@ -1,101 +1,256 @@
...
+
+            <div className="card">
+                <div className="card-body">
+                    <h3 style={{ color: "black" }}>Project details</h3>
+                    <br></br>
+
+                        <div class="container-fluid">
+                            <div className="form-group col-md-5">
+
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove. It shouldn’t be spaces between divs only one at the end of each code element. Just for have in mind :D

File: frontend/src/components/Projects/EditProject.js [hunk 6]
```diff
@@ -1,101 +1,256 @@
...
+                                        className="form-control"
+                                        value={this.state.projectUniqueNumber}
+                                        onChange={e => this.setState({ projectUniqueNumber: e.target.value })}
+                                    />
+                                </div>
+                            </div>
+                            <div className="form-group col-md-5">
+
+
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Same here (Blank space, please remove)

File: frontend/src/components/Projects/EditProject.js [hunk 7]
```diff
@@ -1,101 +1,256 @@
...
+                                        value={this.state.projectName}
+                                        onChange={e => this.setState({ projectName: e.target.value })}
+                                    />
+                                </div>
+                            </div>
+
+                            <div className="form-group col-md-4">
+                                <label>Duration</label>
+
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove

File: frontend/src/components/Projects/EditProject.js [hunk 8]
```diff
@@ -1,101 +1,256 @@
...
+                                    <input
+                                        type="text"
+                                        className="form-control"
+                                        value={this.state.projectName}
+                                        onChange={e => this.setState({ projectName: e.target.value })}
+                                    />
+                                </div>
+                            </div>
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove

File: frontend/src/components/Projects/EditProject.js [hunk 9]
```diff
@@ -1,101 +1,256 @@
...
+                                <label>Budget</label>
+
+
+                                <input
+                                    type="text"
+                                    className="form-control"
+                                    value={this.state.budget}
+                                    onChange={e => this.setState({ budget: e.target.value })}
+                                />
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove

File: frontend/src/components/Projects/EditProject.js [hunk 10]
```diff
@@ -1,101 +1,256 @@
...
+
+                                <input
+                                    type="text"
+                                    className="form-control"
+                                    value={this.state.duration}
+                                    onChange={e => this.setState({ duration: e.target.value })}
+                                />
+
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove

File: frontend/src/components/Projects/EditProject.js [hunk 11]
```diff
@@ -1,101 +1,256 @@
...
+                                <label>Duration</label>
+
+
+                                <input
+                                    type="text"
+                                    className="form-control"
+                                    value={this.state.duration}
+                                    onChange={e => this.setState({ duration: e.target.value })}
+                                />
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove

File: frontend/src/components/Projects/EditProject.js [hunk 12]
```diff
@@ -1,101 +1,256 @@
...
+                                    <input
+                                        type="text"
+
+                                        className="form-control"
+                                        value={this.state.discountRate}
+                                        onChange={e => this.setState({ discountRate: e.target.value })}
+                                    />
+
+                                </div>
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove

File: frontend/src/components/Projects/EditProject.js [hunk 13]
```diff
@@ -1,101 +1,256 @@
...
+                                        className="form-control"
+                                        value={this.state.discountRate}
+                                        onChange={e => this.setState({ discountRate: e.target.value })}
+                                    />
+
+                                </div>
+                            
+                        </div>
+                   
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove

File: frontend/src/components/Projects/MyProject.js [hunk 14]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import ProjectsApi from "./../../api/ProjectsApi";
-import "./MyProjects.css";
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/MyProject.js]: Why removing this? Don't we need this css file?  if don’t then i guess that file should be removed

File: frontend/src/components/Projects/ProjPanel.js [hunk 15]
```diff
@@ -57,7 +53,7 @@ export class ProjPanel extends Component {
 
         >Project Specific Details
 
-        
+
 
         </p>
         <div class="container">
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/ProjPanel.js]: Change "class" to "className"

File: frontend/src/components/Projects/EditProject.js [hunk 16]
```diff
@@ -1,101 +1,256 @@
...
+                                    value={this.state.lastPV}
+                                    onChange={e => this.setState({ lastPV: e.target.value })}
+                                />
+
+                            </div>
+
+                            <div className="form-group col-md-4">
+                                <label>Advanced Payment</label>
+
+
```
Comment (MelanyCristy) [inline frontend/src/components/Projects/EditProject.js]: Blank space, please remove :)

**Review by MelanyCristy — COMMENTED** [review]
I requested a few things for make the cleaner

**Review by pratima-shegokar — COMMENTED** [review]
After clicking edit button 
Screenshot 2019-12-12 at 18 43 07

please solve the issue .

**Comment by rsidhi088** [comment]
Cancelling the pull request because working functionality merging with pull request #81 . The pull request have the same file edits and fetching the values and merged with abed branch .

**Comment by rsidhi088** [comment]
Closing

**Pull request closed without merging**